docs/resourse-cleanup.md

```diff
@@ -0,0 +1,64 @@
+# Stale Resource Cleanup
+
+If the PVC is created with storage class which is having the `reclaimPolicy``
+as `Retain` will not delete the PV object , backend omap metadata and backend image.
+Manual deletion of PV will result in stale omap keys, values , cephfs subvolume and rbd image.
+You need to cleanup metadata and image separately.
+
+## Steps
+
+
+### 1. Get volume ID from PV
+
+* '[$]kubectl get pvc pvc_name -n namespace -owide' to get pv_name <br>
+```
+$ kubectl get pvc mysql-pvc -owide -n prometheus
+NAME        STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE   VOLUMEMODE
+mysql-pvc   Bound    pvc-bc537af8-67fc-4963-99c4-f40b3401686a   20Gi       RWO            csi-rbd        14d   Filesystem
+```
+
+### 2. Get omap keys
+
+* '[$]rados listomapkeys csi.volumes.default -p pool_name | grep pv_name' to get omapkey1 <br>
+```
+$ rados listomapkeys csi.volumes.default -p kube_csi | grep pvc-bc537af8-67fc-4963-99c4-f40b3401686a
```

Comment:
`csi.volumes.default` need not be named with the suffix `default` always. Request a note here to state that the suffix should be the value used for the CLI option [`--instanceid`](https://github.com/ceph/ceph-csi/blob/master/docs/deploy-rbd.md#configuration) in the provisioner (or, nodeplugin daemonset) [deployment](https://github.com/ceph/ceph-csi/blob/2cc59ca41155b21cfcf30999a17e4b029fc2f26c/deploy/rbd/kubernetes/csi-rbdplugin-provisioner.yaml#L105-L111).

Reply:
done